go/src/lib/lomcommon/config.go

```diff
@@ -0,0 +1,309 @@
+package lomcommon
+
+import (
+    "encoding/json"
+    "io/ioutil"
+    "os"
+    "sort"
+)
+
+
+const (
+    Detection string = "Detection"
+    SafetyCheck string = "SafetyCheck"
+    Mitigation string = "Mitigation"
+)
+
+type GlobalConfig_t struct {
+    strings map[string]string
+    ints    map[string]int
+    anyVal  map[string]any
+}
+
+
+func (p *GlobalConfig_t) setDefaults() {
+    p.strings = make(map[string]string)
+    p.ints = make(map[string]int)
+    p.anyVal = make(map[string]any)
+
+    p.ints["MAX_SEQ_TIMEOUT_SECS"] = 120
```

Comment:
Also since the keys are referred in the code, create string constants in this package and use the constant everywhere ?